lib/vector-core/src/transform/mod.rs

```diff
@@ -141,6 +141,23 @@ impl Transform {
 ///   `TaskTransform` or vice versa.
 pub trait FunctionTransform: Send + dyn_clone::DynClone + Sync {
     fn transform(&mut self, output: &mut OutputBuffer, event: Event);
+
+    /// TODO
```

Comment:
TODO left intentionally here and on 152?

Reply:
Oops, will fix.

Reply:
Fixed in a59a99cc9